docs/kpod-diff.1.md

```diff
@@ -13,9 +13,9 @@ Displays changes on a container or image's filesystem.  The container or image w
 
 ## OPTIONS
 
-**--json**
+**--format**
 
-Format output as json
+Alter the output into a different format.  The only valid format for diff is `json`.
```

Comment:
I'd keep these format description lines across the man pages as similar as possible.  Perhaps:
Specify the output format.  The only alternative format for diff is 'json'.